### ISSUE TITLE
Add name to search input

### ISSUE THREAD

**Issue by brentryanjohnson** [ONRR/doi-extractives-data#3306]
Our accessibility audit revealed we're missing an appropriate name for our search input.

```
 • Error: This search input element does not have a name available to an accessibility API. Valid names are: label eleme
nt, title attribute, aria-label attribute, aria-labelledby attribute.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.InputSearch.Name
   ├── #search-input
   └── <input type="search" class="search-box header-nav_search" placeholder="Search" id="search-input" name="q" role="s
earch">
```
